modules/proxy-security-group-rules/main.tf

```diff
@@ -4,13 +4,13 @@
 #
 ###############################################################################
 
-resource "aws_security_group_rule" "allow_proxy_http_inbound" {
```

Comment:
I don't think we need to change the variable names from http to https. I think its better to keep the naming consistent across our modules. Lets just leave these as http.